src/query_results.cpp

```diff
@@ -83,15 +85,15 @@ std::string MongoResultIterator::message() const
   (*next_)["blob_id"].Val(blob_id);
   mongo::BSONObj q = BSON("_id" << blob_id);
   mongo::GridFile f = gfs_->findFile(q);
-  ROS_ASSERT(f.exists());
+  rcpputils::assert_true(f.exists());
   std::stringstream ss(std::ios_base::out);
   f.write(ss);
   return ss.str();
 }
 
 mongo::BSONObj MongoResultIterator::metadataRaw() const
 {
-  ROS_ASSERT(next_);
```

Comment:
Wouldn't this be a use case of `RCLCPP_FATAL_EXPRESSION`?

Reply:
Thanks for pointing this out! Yes, I think `RCLCPP_FATAL_EXPRESSION` is a better option here, which can provide more debug info. Changed to `RCLCPP_FATAL_EXPRESSION`.